tools/pip-requires-test

```diff
@@ -19,3 +19,4 @@ subprocess32==3.5.3; python_version <= '3.0'  # A backport of the subprocess mod
 pycodestyle==2.3.1                            # New package replacing pep8
 mock==2.0.0                                   # Mock package needed for readthedocs build for mocking pycurls
 s3cmd==2.0.2                                  # Used for testing of S3
+python3-saml>=1.6.0						  	  # Used for making SAML requests using python
```

Comment:
Please remove it here and put it directly to the `setup_rucio.py`
There is the possiblity to define extras (Similar to the oracle, kerberos, etc. extras) Please specify one for `saml` with this dependency.

Reply:
Removing this causes the travis test cases to fail as it cannot import the module.